library/src/main/java/org/xmtp/android/library/ApiClient.kt

```diff
@@ -95,7 +95,6 @@ data class GRPCApiClient(
                     "backoffMultiplier" to 2.0,
                     "retryableStatusCodes" to listOf(
                         "UNAVAILABLE",
-                        "CANCELLED",
```

Comment:
O don't think we should retry if it's canceled. That's likely an action taken on purpose.